modules/cnv_upload_virtctl.adoc

```diff
@@ -39,7 +39,7 @@ EOF
 making sure to include your chosen parameters. For example:
 +
 ----
-$ virtctl image-upload --uploadproxy-url=https://$(oc get route cdi-uploadproxy -o=jsonpath='{.status.ingress[0].host}')/v1alpha1/upload --pvc-name=upload-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2
+$ virtctl image-upload --uploadproxy-url=https://cdi-uploadproxy-kube-system.apps.10.19.138.91.nip.io --pvc-name=upload-fedora-pvc --pvc-size=5Gi --image-path=tmp/Fedora-Cloud-Base-29-1.2.x86_64.qcow2
```

Comment:
The `--uploadproxy-url` would be better set to `https://$(oc get route cdi-uploadproxy -o=jsonpath='{.status.ingress[0].host}')` without any path (`/v1alpha1/upload`) or trailing slashes.

Also, the `image-path` can be kept as it originally was, that wasn't the issue.